### PR TITLE
🌐 [i18n-ZH] Translate multilingual into Chinese

### PR DESCRIPTION
**Pull request by yyLeaves**
# What does this PR do?
Add zh (Chinese) translation for multilingual.md. #20095 
## Who can review?
Documentation: @stevhliu 
Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

File: docs/source/zh/multilingual.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+- `xlm-mlm-enfr-1024` （掩码语言建模，英语-法语）
+- `xlm-mlm-enro-1024` （掩码语言建模，英语-罗马尼亚语）
+- `xlm-mlm-xnli15-1024` （掩码语言建模，XNLI 数据集语言）
+- `xlm-mlm-tlm-xnli15-1024` （掩码语言建模+翻译，XNLI 数据集语言）
+- `xlm-clm-enfr-1024` （因果语言建模，英语-法语）
+- `xlm-clm-ende-1024` （因果语言建模，英语-德语）
+
+语言嵌入被表示一个张量，其形状与传递给模型的 `input_ids` 相同。这些张量中的值取决于所使用的语言，并由分词器的 `lang2id` 和 `id2lang`  属性识别。
+
+在此示例中，加载 `xlm-clm-enfr-1024` 检查点（因果语言建模，英语-法语）：
```
Comment (stevhliu) [inline docs/source/zh/multilingual.md:41]:
```suggestion
在此示例中，加载 `xlm-clm-enfr-1024` 检查点（因果语言建模，英语-法语)：
```

**Review by stevhliu — APPROVED** [review]
Great job! 🤗

File: docs/source/zh/multilingual.md (hunk 1 above)
Comment (yyLeaves) [reply docs/source/zh/multilingual.md:41]: In my opinion, change the right bracket alone to English style makes it unmatch with the left bracket.

Comment (stevhliu) [reply docs/source/zh/multilingual.md:41]: Ah ok, so we should keep the space between the `)` and `:`? For example, on line 29 there is no space between the `:` and the last character.

Comment (yyLeaves) [reply docs/source/zh/multilingual.md:41]: > Ah ok, so we should keep the space between the `)` and `:`? For example, on line 29 there is no space between the `:` and the last character.

I don't think there is extra space between `）` and `：`. The problem is that they are Chinese punctuations and look like they take one extra space but actually there isn't a space in between, as in line 41. 
- English punctuation: `):`
- Chinese punctuation: `）：`

Some subtitle group such as 人人字幕组 don't like it either, so they replace some punctuations in Chinese subtitles into English style to make it look better.

Comment (stevhliu) [reply docs/source/zh/multilingual.md:41]: Thanks for the clarification! In that case, we're good to merge :)

Comment (yyLeaves) [reply docs/source/zh/multilingual.md:41]: @stevhliu Thanks!

**Pull request merged**